applications/GeoMechanicsApplication/custom_utilities/hydraulic_discharge.h

```diff
@@ -0,0 +1,32 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Anne van de Graaf
+//
+
+#pragma once
+
+#include "custom_utilities/node_utilities.h"
+#include "geo_mechanics_application_variables.h"
+#include "includes/kratos_export_api.h"
+#include "includes/properties.h"
```

Comment:
Not all includes (e.g. for node utilities) are needed in the header, so they can be moved to the source file. However, there are also a few which are included implicitly via other headers (I think it's always better to be explicit). I think this would be a complete list without redundancies:
```suggestion
#include "containers/array_1d.h"
#include "geometries/geometry.h"
#include "geometries/geometry_data.h"
#include "includes/kratos_export_api.h"
#include "includes/node.h"

#include <vector>
```

Reply:
done